recipes/gdbm/all/conandata.yml

```diff
@@ -5,3 +5,9 @@ sources:
       "https://mirrors.tripadvisor.com/gnu/gdbm/gdbm-1.18.1.tar.gz",
     ]
     sha256: "86e613527e5dba544e73208f42b78b7c022d4fa5a6d5498bf18c8d6f745b91dc"
+  "1.19":
+    url: [
+      "https://ftp.gnu.org/gnu/gdbm/gdbm-1.19.tar.gz",
+      "https://mirrors.tripadvisor.com/gnu/gdbm/gdbm-1.19.tar.gz",
+    ]
+    sha256: "37ed12214122b972e18a0d94995039e57748191939ef74115b1d41d8811364bc"
```

Comment:
LGTM, but could you follow the same ordering than in `config.yml` for consistency please?

Reply:
Changed the order in the `config.yml`. At least openssl recipe use ascending order for version.

Reply:
there are no guidelines regarding ascending/descending order (I prefer descending order), so it's fine for me.